submissions/factorio_draftsman/main.md

```diff
@@ -0,0 +1,200 @@
+# Introduction
+
+A couple months ago, I decided that I wanted to try my hand at making a **Self-expanding factory** in Factorio. After seeing an number of [impressive](https://www.youtube.com/watch?v=xF--1XdcOeM) [examples](https://www.youtube.com/watch?v=KNSBerFe_H8), I was inspired to have a go at the problem. I already had an outline of the logic, and how the factory would keep track of itself, as well as lofty ideas on impressive things that I could make it do. The only trouble was that I had actually never used combinators before, and I was planning on using them for the actual decision-making.
```

Comment:
Maybe instead of referencing the JOSEF video you can reference the article about it :)
https://alt-f4.blog/ALTF4-39/

Reply:
Done. Forgot that you guys already had an article on it.